### ISSUE TITLE
Problem 262 - MS SQL Server, usage of temp table variable yields 'Runtime error'

### ISSUE THREAD

**Issue by Kri-Sha** [LeetCode-Feedback/LeetCode-Feedback#2203]
#### Your LeetCode username

 Kri_Sha

#### Category of the bug
- [] Question
- [X] Solution
- [ ] Language

#### Description of the bug

The environment doesn't seem to support usage of temp table variable in the solution, yields 'Runtime error'. The same code runs fine on MS SQL Server 2019 on my machine.

#### Code you used for Submit/Run operation

```
declare @dates as table(Request_at date);
declare @temp as date = '2013-10-01';
insert into @dates 
    values (@temp);
select * from @dates
```

#### Language used for code

MS SQL Server

#### Expected behavior

Running the same code in MS SQL Server 2019 returns the contents of the temp table. I expect the same behavior here.
When trying to copy the error contents, nothing is copied, i.e. no way to know what is going on.

#### Screenshots

![image](https://user-images.githubusercontent.com/25306925/106372272-e9f33480-6322-11eb-96a5-c8b082943244.png)

**Comment by lc-kay** [comment]
@Kri-Sha Here is the Runtime Error message I received when I ran your code, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'declare @dates as table(Request_at date);
declare @temp as date = '2013-10-01';
' at line 2" I hope this clarifies any doubts or concerns. Thank you for your time.

**Comment by Kri-Sha** [comment]
@lc-kay , my code is for MS Sql, not MY Sql. 
Please, check it again.

**Comment by lc-kay** [comment]
@Kri-Sha I apologize for the mistake. I checked it again and was able to reproduce the issue. I've relayed this issue to our team to investigate. However, upon investigating this issue, we found this issue to be non-content-related (the issue stems from an origin away from the content on this problem); therefore, it's not valid for content contribution reward. 
We encourage you to share more content-related issues here with us for [valid content contribution rewards](https://leetcode.com/discuss/general-discussion/526802/File-Problem-and-Solution-Content-Issues-to-Our-GitHub-Public-Feedback-Repo-Now!).
